airbyte-integrations/connectors/source-dremio/integration_tests/configured_catalog.json

```diff
@@ -0,0 +1,14 @@
+{
+  "streams": [
+    {
+      "stream": {
+        "name": "catalogs",
+        "source_defined_primary_key": [["id"]],
+        "json_schema": {},
+        "supported_sync_modes": ["full_refresh"]
+      },
+      "sync_mode": "full_refresh",
+      "destination_sync_mode": "overwrite"
```

Comment:
the primary key is missing

Reply:
primary key is defined in `source_defined_primary_key` on line 6, or should there be a different name for this field?